src/components/ListItem.jsx

```diff
@@ -55,8 +74,13 @@ export function ListItem({ item, listPath }) {
 		return;
 	};
 
+	const UrgencyStatusIcon = urgencyStatusIcons[itemUrgencyStatus];
+
 	return (
-		<li className="ListItem">
+		<li className={`ListItem`}>
+			{UrgencyStatusIcon && (
+				<UrgencyStatusIcon style={urgencyStatusWidth} fontSize="large" />
```

Comment:
I would suggest having a "tooltip" on hover of these icons. I didn't see a key anywhere to indicate what the icons mean on the app itself, so having a tooltip would help indicate by text what the icon means.

I would also consider adding an `aria-label` to indicate helper text on what the icon means as well.

Reply:
Thank you! Will add Tooltip component from Material, as well as aria-labels!